open_ce/build_tree.py

```diff
@@ -344,16 +344,23 @@ def _create_remote_deps(self, dep_graph):
                 node = deps.pop()
                 ancestor_build_cmds = {x.build_command for x in networkx.ancestors(dep_graph, node)
                                                                 if x.build_command is not None}
+                channels = node.channels
+
                 ancestor_channels = []
                 for cmd in ancestor_build_cmds:
                     ancestor_channels += cmd.channels
+
+                for channel in ancestor_channels + self._channels:
```

Comment:
This assumes that there are no duplicates in `node.channels`.

Should we assign channels to an empty list above, and have channel iterate through `node.channels + ancestor_channels + self._channels`?

Reply:
Yeah, that would work as well. I wasn't as worried about that because I think it would have to be added twice within a config file.